xml/obs_build_config.xml

```diff
@@ -398,6 +398,19 @@ Keep: debbuild</screen>
                 build results get published immediately after the build is finished.</para>
         </listitem>
        </varlistentry>
+       <varlistentry>
+        <term><parameter>nofailedpackages</parameter> (OBS 2.11 or later)</term>
+        <listitem>
+           <para>Block publishing if any build result was failed, broken or unresolvable. Only consistent succeeding builds 
+                 get published. Please note that this gets only evaluated per scheduler architecture.</para>
+        </listitem>
+       </varlistentry>
+       <varlistentry>
+        <term><parameter>withreports</parameter> (OBS 2.11 or later)</term>
+        <listitem>
+          <para>Also publish internal content tracking files (<literal>.report</report> files).</para>
```

Comment:
This change introduces a minor validation issue:

```suggestion
          <para>Also publish internal content tracking files (<literal>.report</literal> files).</para>
```

Reply:
fixed